build-scripts/travis.sh

```diff
@@ -483,20 +388,23 @@ test_auth() {
   run "test -f modules/remotebackend/test-suite.log && cat modules/remotebackend/test-suite.log || true"
 
   #DNSName - make -k -j3 -C pdns $(grep '(EXEEXT):' pdns/Makefile | cut -f1 -d\$ | grep -E -v 'dnsdist|calidns')
-  run 'make -k -j3 -C pdns $(grep "(EXEEXT):" pdns/Makefile | cut -f1 -d\$)'
+  run 'make -k -j3 -C pdns $(grep "(EXEEXT):" pdns/Makefile | cut -f1 -d\$ | grep -v comfun)'
 
   run "cd pdns"
   run "./pdnsutil test-algorithms"
   run "cd .."
 
+  run "sudo systemctl start slapd.service mysql.service postgresql.service"
   run "cd regression-tests"
 
   #travis unbound is too old for this test (unbound 1.6.0 required)
   run "touch tests/ent-asterisk/fail.nsec"
 
+  run "sudo systemctl start slapd.service"
```

Comment:
only start services when we need them (this frees up some memory at the expense of paying to start/stop)